plugins/kotlin/uast/uast-kotlin/tests/testData/ClassAnnotation.render.txt

```diff
@@ -7,7 +7,7 @@ public abstract annotation MyAnnotation {
 }
 
 public final class B {
-    @null public static final var Companion: B.Companion
+    @null @MyAnnotation(text = "companion") public static final var Companion: B.Companion
```

Comment:
So, these two (below too) are a bug in ULC / K1, resulting in some test data split (e.g., `plugins/kotlin/uast/uast-kotlin-fir/tests/testData/legacyValues/ClassAnnotation.\S+.fe10.txt`).

Also, their anchor parent chains (from that annotation) become wrong. Unlike K2 UAST tests where we can control expected(?) exceptions, K1 UAST tests don't. Since they're technically duplicates, I removed some of K1 UAST tests.